Terminal Response

xia0 opened the pull request:
just adding my humble plugin

OllisGit commented:
Some remarks:
- Currently I have no idea why the Build-Failed. In the past a restart of the build solve the issue.
Someone from the experienced developer should take a look on this.

- No need to push these file/folder to git: ```.pyc``` and ```.egg-info```.

Just create a ```.gitignore```-file that contains something like this:
```
*.pyc
*.swp
.idea
*.iml
build
dist
*.egg*
.DS_Store
*.zip
```
- Everything else looks fine

xia0 commented:
> 
> 
> Some remarks:
> 
>     * Currently I have no idea why the Build-Failed. In the past a restart of the build solve the issue.
>       Someone from the experienced developer should take a look on this.
> 
>     * No need to push these file/folder to git: `.pyc` and `.egg-info`.
> 
> 
> Just create a `.gitignore`-file that contains something like this:
> 
> ```
> *.pyc
> *.swp
> .idea
> *.iml
> build
> dist
> *.egg*
> .DS_Store
> *.zip
> ```
> 
>     * Everything else looks fine:
>       ![image](https://user-images.githubusercontent.com/24205767/80400134-a7e02280-88ba-11ea-8182-61616fea2723.png)

Thanks for reviewing. I've added the .gitignore and removed those files.
